src/Exception/InvalidParameterException.php

```diff
@@ -13,11 +13,6 @@
 
 namespace Sonata\NotificationBundle\Exception;
 
-use Sonata\CoreBundle\Exception\InvalidParameterException as CoreBundleException;
-
-/**
- * @deprecated Use Sonata\CoreBundle\Exception\InvalidParameterException instead
- */
-class InvalidParameterException extends CoreBundleException
+class InvalidParameterException extends \RuntimeException
```

Comment:
I think you have to do the same trick as you did for other classes

`if class exist, extends Sonata\CoreBundle\Exception\InvalidParameterException, else extends Runtime`.

Because someone could catch Sonata\CoreBundle\Exception\InvalidParameterException and then this exception would not be catch anymore.

Reply:
done